content/general/concepts/semantic-technologies/semantic-technologies.md

```diff
@@ -0,0 +1,82 @@
+---
+Title: 'Semantic Technologies'
+Description: 'A class of technologies used for data management with a focus on the underlying meaning of the data.'
+Subjects:
+  - 'Data Science'
+Tags:
+  - 'Data'
+  - 'Database'
+  - 'Graphs'
+  - 'Metadata'
+  - 'Semantic'
+CatalogContent:
+  - 'paths/data-science-foundations'
+  - 'paths/data-science-nlp'
+---
+
+**Semantic technologies** are a class of technologies used for data management. Compared to other data management technologies, semantic technologies have a particular focus on making the underlying meaning of the data more transparent to both human and machine users of that data.
+
+Semantic technologies encompass a variety of standards, frameworks, data models, and applications, which are described in further detail below.
+
+## Semantic Data Models
+
+Semantic technologies organize information using the following data models.
+
+### Taxonomies
+
+A **taxonomy** consists of heirarchies of concepts. For example, the concept `poodle` is a subclass of the concept `dog`, in the sense that all poodles are dogs. In turn, the concept `dog` is a subclass of the concept `mammal`. Taxonomies are data models consisting of definitions of concepts and a specification of the heirarchies (relations of subclass and superclass) between them.
+
+### Ontologies
+
+An **ontology** is an enriched taxonomy. Ontologies include all the elements in taxonomies and add the following:
+
+- *Additional relations over concepts*: Ontologies can include relations other than subclass and superclass. For example, to represent the information that cars have tires as parts, one can define a relation `partOf` as holding between the concepts `tire` and `car`. Notice that this is not a heirarchy relation: it does not say that all tires are cars or that all cars are tires.
+- *Properties of concepts*: Ontologies can include properties that apply to concepts. For example, to represent the information that cars have weights, one can define a property `weight` as applying to the concept `car`.
+- *Additional information about properties and relations*
+  - *The domain and range of properties and relations*: Ontologies can include specifications of what types of entities a property or relation applies to (domain) and what values it can take (range). For example, one could specify that the domain of a relation `authorOf` is `person`, and the range is `document`.
```

Comment:
```suggestion
  - The domain and range of properties and relations*: Ontologies can include specifications of what types of entities a property or relation applies to (domain) and what values it can take (range). For example, one could specify that the domain of a relation `authorOf` is `person`, and the range is `document`.
```